scripts/v2/scripts.js

```diff
@@ -35,7 +35,7 @@ function sampleRUM(checkpoint, data = {}) {
     const { random, weight, id } = window.hlx.rum;
     if (random && (random * weight < 1)) {
       // eslint-disable-next-line object-curly-newline
-      const body = JSON.stringify({ weight, id, referer: window.location.href, generation: 'blog-gen2', checkpoint, ...data });
+      const body = JSON.stringify({ weight, id, referer: window.location.href, generation: 'blog-gen3', checkpoint, ...data });
```

Comment:
```suggestion
      const body = JSON.stringify({ weight, id, referer: window.location.href, generation: 'blog-martech-sync', checkpoint, ...data });
```

That makes it a bit easier to interpret the results without a mapping table.